### PR TITLE
feat: Sticky Strategies Nav for Desktop

### PR DESCRIPTION
**Pull request by allozaur**
### Video demo

https://user-images.githubusercontent.com/37627284/207640567-8d290ae6-2d47-406b-a648-cb06b500ea66.mov

### CODE REVIEW

File: src/routes/+layout.svelte [hunk 1]
```diff
@@ -56,7 +56,7 @@
 	 */
 	@custom-media --viewport-sm-up (width >= 576px);
 	@custom-media --viewport-md-up (width >= 768px);
-	@custom-media --viewport-lg-up (width >= 1024px);
+	@custom-media --viewport-lg-up (width >= 1025px);
```
Comment (kenkunz) [inline src/routes/+layout.svelte:59]: Hmm… this doesn't seem right to me.

I don't have an opinion or preference about which direction (`up` vs. `down`) to be _inclusive_ vs. _exclusive_, but I think we need be consistent. So if we want to be _inclusive_ on the `up` side of the breakpoints, then I'd recommend we go with:

```css
@custom-media --viewport-sm-up (width > 576px);
@custom-media --viewport-md-up (width > 768px);
@custom-media --viewport-lg-up (width > 1024px);
@custom-media --viewport-md-down (width <= 1024px);
@custom-media --viewport-sm-down (width <= 768px);
@custom-media --viewport-xs (width <= 576px);
```

**Review by kenkunz — CHANGES_REQUESTED** [review]
I think this is a beneficial UX improvement!

Just have one concern about the change to the media query breakpoints.

File: src/routes/+layout.svelte (hunk 1 above)
Comment (allozaur) [reply src/routes/+layout.svelte:59]: @kenkunz yes, sure, we can do it like this!

**Review by kenkunz — APPROVED** [review]
lgtm!

**Comment by miohtama** [comment]
Ship it 👍

**Pull request merged**